foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/enable-habushu-build-cache-migration.feature

```diff
@@ -0,0 +1,17 @@
+@habushu-build-cache-migration
+Feature: Adds the <build><directory>dist</directory></build> tag to the pom file if not present.
+
+  Scenario: Migrate a pom file
+    Given a habushu-packaged pom file where the build directory is unspecified
+    When the migration executes
```

Comment:
S: would be nice if this is a bit more specific since steps conflict across files.  Have witnessed this cause many developers to lose hours not realizing their step was already defined in another step class in the past.

Reply:
New version: the habushu build cache migration executes